spec/lib/saml_idp/metadata_builder_spec.rb

```diff
@@ -33,7 +33,7 @@ module SamlIdp
 
     it "includes logout element" do
       subject.configurator.single_logout_service_post_location = 'https://example.com/saml/logout'
-      expect(subject.fresh).to match(
+      expect(subject.fresh).not_to match(
         '<SingleLogoutService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://example.com/saml/logout"/>'
```

Comment:
I would shorten this to just making sure SingleLogoutService isn't there, rather than the full thing, because it could be included with different params which is not the goal

Reply:
Done